composer.json

```diff
@@ -12,7 +12,7 @@
 	"require": {
 		"php": ">=5.3.0",
 		"wikibase/data-model": "~4.1",
-		"wikibase/data-model-services": "~1.1"
+		"wikibase/data-model-services": "~2.0"
```

Comment:
Actually, this could have been `~1.1|~2.0`. Is it ok for you if I do change this manually?